test/test_models.py

```diff
@@ -6,10 +6,17 @@
 
 import unittest
 import torch
+from torch import nn
 import numpy as np
 import random
 from pyrovision import models
 from torchvision.models.resnet import BasicBlock
+from PIL import Image
+from pathlib import Path
+from torchvision import transforms
+
+# Define fire image to test models on a real use case
+testImage = Path(__file__).parent / 'fixtures/wildfire_example.jpg'
```

Comment:
This is specific to the test you added, I suggest moving it inside it if we don't use it for other unittests